src/r5py/util/validatingrequestssession.py

```diff
@@ -54,9 +54,7 @@ def request(self, method, url, checksum, **kwargs):
 
         if digest != checksum:
             raise ChecksumFailed(
-                ("Checksum failed for {}" + "expected {}, got {}").format(
-                    url, checksum, digest
-                )
+                f"Checksum failed for {url}" + "expected {checksum}, got {digest}"
             )
```

Comment:
This won't produce expected text. The correct syntax would be:
```python
f"Checksum failed for {url}, expected {checksum}, got {digest}."
```

Reply:
Good spot! How did I miss this?